config/data/langs.toml

```diff
@@ -479,8 +479,8 @@ void main(List<String> argv) {
 [Elixir]
 args    = [ '/usr/local/bin/elixir', '-e', '$code', '--' ]
 env     = [ 'LANG=C.UTF-8', 'PATH=/usr/local/bin:/usr/bin:/bin' ]
-size    = '158 MiB'
-version = '1.16.3'
+size    = '178 MiB'
```

Comment:
Well I guess +20 megs isn't quite as bad as +164 megs :sweat_smile:

Reply:
> Well I guess +20 megs isn't quite as bad as +164 megs 😅

Yeah, man. This is acceptable. Not so long ago, that day when I bumped like 30%ish of the langs, the difference was still in the hundreds.